Documentation/README.md

```diff
@@ -0,0 +1,19 @@
+# Documentation
+
+This folder contains the documents that are linked in the [README](../README.md).
+
+## Demos
+
+- [Demo: the "execsnoop" gadget](demo-execsnoop.md) – watch it [as GIF](images/demo-execsnoop-gifterminal.md)
```

Comment:
Can we move all `demo-*.md` to `demos` subdirectory and remove `demo-` prefix then?

Reply:
I wouldn't hide them away in subdirectories.